### PR TITLE
finding todos by id done with validations and error handling

### PR DESCRIPTION
**Pull request by anita210**
-command switch case in app.js
-findbyid in todo
-validations for id input and possible no result for id search

### CODE REVIEW

File: src/todo.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
+
 export function format(todo) {
-  return `${todo.id} - [${todo.done ? '✅': ' '}] ${todo.title}`;
+  return `${todo.id} - [${todo.done ? '✅' : ' '}] ${todo.title}`;
```
Comment (ZsoltEmber) [inline src/todo.js:3]: Do we want x or ✅? I went with x

File: src/todo.spec.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
-import { jest } from '@jest/globals';
-import { add, format, formatList, list } from './todo.js';
+import { jest } from '@jest/globals'; import { add, findById, format, formatList, list } from './todo.js';
```
Comment (ZsoltEmber) [inline src/todo.spec.js:1]: This should be separated

**Review by ZsoltEmber — COMMENTED** [review]
Lgtm

File: src/todo.js (hunk 1 above)
Comment (anita210) [reply src/todo.js:3]: x is fine to me, i saw, that you fixed that

File: src/todo.spec.js (hunk 2 above)
Comment (anita210) [reply src/todo.spec.js:1]: true true didnt mean to put that in one line

**Pull request merged**